aten/src/ATen/native_parse.py

```diff
@@ -11,19 +11,49 @@
     from yaml import Loader
 
 
+# [temp translations]
+# We're currently incrementally moving from the custom func schema to the
+# JIT signature schema incrementally. This will reduce overall complexity
+# and increase compliance between these components. So for now we do simple
+# type translations to continue to emit the legacy func schema for further
+# processing by downstream tools. This will helps us avoid having to prematurely
+# change all downstream tools to detect these new types.
+def temp_type_translations(typ):
+    # Enables Tensor[] by translating to legacy TensorList. See [temp translations]
+    if typ == 'Tensor[]':
+        return 'TensorList'
+    # Enables int[] by translating to legacy IntList. See [temp translations]
+    if typ == 'int[]':
+        return 'IntList'
+    # Enables int by translating to legacy int64_t. See [temp translations]
+    if typ == 'int':
+        return 'int64_t'
+    # Enables float by translating to legacy double. See [temp translations]
+    if typ == 'float':
+        return 'double'
+    return typ
+
+
 def parse_default(s):
     if s.lower() == 'true':
         return True
     elif s.lower() == 'false':
         return False
     elif s == 'nullptr':
         return s
-    elif s == '{}':
+    # Enables default argument [] by translating to legacy {}.
+    # See [temp translations]
+    elif s == '[]':
         return '{}'
-    elif re.match(r'{.*}', s):
-        return s
+    elif re.match(r'\[.*\]', s):
+        return "{" + s[1:-1] + "}"
```

Comment:
This is not an actionable comment, but it's kind of irritating the temp translations have to be spattered all over the code. Good thing they're labeled `[temp translations]`